### PR TITLE
Category

### PR DESCRIPTION
**Pull request by mimansha-swarup**
image

image

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
Looks good!

File: src/Components/Home/CategoryCard.tsx [hunk 1]
```diff
@@ -1,17 +1,21 @@
 import { Link } from "react-router-dom";
 import React from "react";
-
-export const CategoryCard: React.FC = () => {
+import { DocumentData } from 'firebase/firestore';
+interface CategoryProps {
+  data: DocumentData,
+}
+export const CategoryCard:React.FC<CategoryProps> = ({data}) => {
+  // const {name,coverImage} = data
```
Comment (gopalbbharadva) [inline src/Components/Home/CategoryCard.tsx]: Can remove comment.

File: src/Pages/Category/CategoryPage.tsx [hunk 2]
```diff
@@ -1,14 +1,51 @@
...
+            type: "SET_QUIZ",
+            payload: { id: quizDoc.id, data: quizDoc.data() },
+          });
+          setQuizzesList((prev) => [...prev, quizDoc.data()]);
+        }
+      });
+    })();
+  }, [categoryName, categories]);
+  console.log(quizzesList)
+  console.log("quizzesList")
```
Comment (gopalbbharadva) [inline src/Pages/Category/CategoryPage.tsx]: REmove unsued consoles.

**Review by gopalbbharadva — APPROVED** [review]
Sign up is showing firebase api key error. check once.

**Pull request merged**